### PR TITLE
PHP 8.0 Windows support

### PR DESCRIPTION
**Pull request by pepone**
This PR adds Windows support for PHP 8.0, and fixes some incorrect usages of ` ice_void_arginfo` used with methods that take parameters.

### CODE REVIEW

File: php/src/php/Operation.cpp [hunk 1]
```diff
@@ -383,15 +383,15 @@ IcePHP::OperationI::getArgInfo(zend_internal_arg_info& arg, const ParamInfoPtr&
     {
         zend_internal_arg_info ai[] =
         {
-            ZEND_ARG_ARRAY_INFO(pass_by_ref, 0, allow_null)
+            ZEND_ARG_ARRAY_INFO((uint32_t)pass_by_ref, (uint32_t)0, (uint32_t)allow_null)
```
Comment (externl) [inline php/src/php/Operation.cpp:386]: Will this work with php < 8?

Comment (externl) [inline php/src/php/Operation.cpp:386]: I think we usually prefer static cast over c-style casts.

File: php/src/php/Operation.cpp [hunk 2]
```diff
@@ -383,15 +383,15 @@ IcePHP::OperationI::getArgInfo(zend_internal_arg_info& arg, const ParamInfoPtr&
...
         };
         arg = ai[0];
     }
     else
     {
         zend_internal_arg_info ai[] =
         {
-            ZEND_ARG_CALLABLE_INFO(pass_by_ref, 0, allow_null)
+            ZEND_ARG_CALLABLE_INFO((uint32_t)pass_by_ref, (uint32_t)0, allow_null)
```
Comment (externl) [inline php/src/php/Operation.cpp:394]: Ditto

File: scripts/Util.py [hunk 3]
```diff
@@ -3784,7 +3784,7 @@ def getSupportedArgs(self):
         def usage(self):
             print("")
             print("PHP Mapping options:")
-            print("--php-version=[7.1|7.2|7.3]    PHP Version used for Windows builds")
+            print("--php-version=[7.1|7.2|7.3|8.0]    PHP Version used for Windows builds")
```
Comment (externl) [inline scripts/Util.py:3787]: Do we support 8.1 too?

**Review by externl — COMMENTED** [review]
Looks good! Just a few comments. Should we also do this change in for php5?

File: php/src/php/Operation.cpp (hunk 1 above)
Comment (pepone) [reply php/src/php/Operation.cpp:386]: I think so need to check

Comment (pepone) [reply php/src/php/Operation.cpp:386]: right I will fix

File: scripts/Util.py (hunk 3 above)
Comment (pepone) [reply scripts/Util.py:3787]: Need to test 8.1, I didn't yet

File: php/src/php/Operation.cpp [hunk 4]
```diff
@@ -174,7 +174,7 @@ IcePHP::ResultCallback::ResultCallback()
 
 IcePHP::ResultCallback::~ResultCallback()
 {
-    zval_ptr_dtor(&zv);
+
```
Comment (externl) [inline php/src/php/Operation.cpp:177]: Extra newline here?

**Review by externl — APPROVED** [review]

**Pull request merged**